sql/src/main/antlr/OpenSearchSQLParser.g4

```diff
@@ -323,6 +324,19 @@ getFormatType
     | TIMESTAMP
     ;
 
+nestedFunction
+    : NESTED LR_BRACKET
+    (nestedField | nestedField COMMA nestedPath | nestedPath COMMA expression)
```

Comment:
Are the last two cases relevant for SELECT? If not, maybe remove for now until WHERE implementation?